### ISSUE TITLE
Excessive logging when using forwarding / missing url info in case of error

### ISSUE THREAD

**Issue by mcm1957** [phifogg/ioBroker.sainlogic#168]
Please consider changing the logging level for successful forwarded messages to DEBUG.
Currently every update is logged as info:

![image](https://github.com/phifogg/ioBroker.sainlogic/assets/28575778/503b07da-075c-4914-981f-d5174753a47a)

It's OK to log problems; it would be ok to log the success ONCE after starting the adapter - but loggin an info for every data transmitted ist a little bit to much in my oppinion.

In opposite any problem transmitting the data is also logged with info level. This could be kept or alternatively raised to warning. Bute while logging the urls string used whenever the operation is ok this info (the url string used) is NOT logged whenever an error occures. It would be much more useful in those cases-.

```
2023-06-16 15:38:09.986 - info: sainlogic.0 (20550) State value to set for "sainlogic.0.weather.current.hourlyrain_piezo" has to be type "number" but received type "string"
--
2023-06-16 15:38:26.452 - info: sainlogic.0 (20550) HTTPError: Response code 401 (Unauthorized)
2023-06-16 15:38:55.625 - info: admin.0 (2141) ==> Connected system.user.admin from ::ffff:10.17.2.110
2023-06-16 15:38:58.788 - info: sainlogic.0 (20550) HTTPError: Response code 401 (Unauthorized)
2023-06-16 15:39:13.900 - info: host.pi4 stopInstance system.adapter.sainlogic.0 (force=false, process=true)
```

THANKS for the adapter !

Adapter Version: 0.10.3

**Comment by phifogg** [comment]
Moved messages to debug level.

**Comment by phifogg** [comment]
Closed with 0.10.5
